openslides/users/static/js/users/site.js

```diff
@@ -686,10 +688,27 @@ angular.module('OpenSlidesApp.users.site', [
 
         // Export as PDF
         $scope.pdfExportUserList = function () {
-            var filename = gettextCatalog.getString("List of participants")+".pdf";
-            var userListContentProvider = UserListContentProvider.createInstance($scope.usersFiltered, $scope.groups);
-            var documentProvider = PdfMakeDocumentProvider.createInstance(userListContentProvider);
-            PdfCreate.download(documentProvider.getDocument(), filename);
+            var imageMap = {};
+            var imagePromises = [];
+            var imageSources = [];
+
+            imageSources.push(Config.get('logo_pdf_header').value.path);
+            imageSources.push(Config.get('logo_pdf_footer').value.path);
+
+            imagePromises = _.map(imageSources, function (image_source) {
```

Comment:
This was only quick hadded into user list to try out the logo features... and @emanuelschuetze wanted to see some progress ;)

However, you are right, now every PDF needs async Image transformation, and we probably gonna need something like a parent for our conten providers.

We might even need to separate more, since logos are a document provider related (the logo has nothing to do with the content of e.g. the user list but with the overall design of the document) and motion images (and possibly other images as well) are content provider related. We might want to keep that separate.